src/ERC4973Permit.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
This repository seems to use a different tabwith/number of spaces compared to the other .sol files. I've created #9. We probably also want to run a CI workflow that ensures proper number of indentation.